src/AddOn.Episerver.Settings/Core/SettingsService.cs

```diff
@@ -191,11 +198,11 @@ public IEnumerable<ContentReference> SettingsRoots
     }
 
     /// <summary>
-    ///     Gets the settings.
+    ///     Gets the setting.
```

Comment:
Gets the setting given the specified type from the global settings repository.

Reply:
Fixed